### PR TITLE
CUMULUS-2920 -- Update Core npm build version to 8.6.4

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

Addresses [CUMULUS-2920: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2920)

## Changes

* This PR updates npm to 8.6.4, and addresses a failed dependency concern.   This is intended specifically to allow us to use https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides in case of dependency concerns as a stopgap. 

## PR Checklist

- [x] Update CHANGELOG
- [ ] Unit tests
- [x] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: bamboo/Dockerfile [hunk 1]
```diff
@@ -8,5 +8,5 @@ xz-utils tk-dev libffi-dev liblzma-dev python3-pip
 RUN update-alternatives --install /usr/bin/python python /usr/bin/python3.7 1 && \
     update-alternatives --install /usr/bin/pip pip /usr/bin/pip3 1
 RUN python -m pip install -U pip
-RUN npm install -g npm@6.14 && npm config --global set script-shell /bin/bash
+RUN npm install -g npm@8.6.0 && npm config --global set script-shell /bin/bash
```
Comment (kkelly51) [inline bamboo/Dockerfile:11]: Do we want to keep the patch version off like before? What was the thought of adding it in?

Comment (Jkovarik) [reply bamboo/Dockerfile:11]: Hmm, that's fair.    I was erring on the side of being explicit, but there's no reason we can't take patches by default. .

Comment (Jkovarik) [reply bamboo/Dockerfile]: 419d186

**Review by charleshuang80 — APPROVED** [review]

**Comment by charleshuang80** [comment]
When I started installing everything, I accidentally upgraded my npm version to 8.6.x since I was prompted to do so. This led to a few things not installing properly, so I had to downgrade back to 6.14.6. I assume you're not seeing any similar issues with this change...

**Comment by Jkovarik** [comment]
@charleshuang80  I did, prior to the eslint plugin update.   npm v8 does better broken cross-dependencies and that change caused a bunch of cascading issues in the base package install.

**Pull request merged**